helm_deploy/values-dev.yaml

```diff
@@ -12,6 +12,7 @@ generic-service:
     PRISON_REGISTER_API_URL: "https://prison-register-dev.hmpps.service.justice.gov.uk"
     SUPPORTED_PRISONS: ""
     COVERSHEET_PRINT_DEBUG: "true"
+    LSJ_SESSION_DURATION_IN_DAYS: 1
```

Comment:
Our session duration is an unbelievable 7 days in prod !!! (don't ask !!!) - in dev we set it at 1 day to facilitate testing etc

Reply:
Horrible - I'm so sorry :(